python/ray/serve/api.py

```diff
@@ -718,3 +726,103 @@ def my_deployment_function(request):
     """
     _request_context = ray.serve.context._serve_request_context.get()
     return _request_context.multiplexed_model_id
+
+
+@PublicAPI(stability="alpha")
+def get_app_handle(
+    name: str, sync: Optional[bool] = None
```

Comment:
What do you think about setting `name` to `"default"` by default? We do that when people deploy an app, so it might make sense to do it when they retrieve the handle.

Reply:
Hm, I'm a little hesitant to do that because when people deploy an app, there is still a required positional argument which is the application (e.g. `serve.run(f.bind())`). If we set a default for `name` here, it might be confusing for users to call `serve.get_app_handle()` without any arguments. I'm open to it, though. What do other folks think? @sihanwang41 @edoakes @GeneDer

Reply:
i don't have strong opinion for this. Both are okay to me. We can start from required argument first and then if user are confused to get the "default" name, we can add it later.